src/commands/init.ts

```diff
@@ -106,7 +120,7 @@ export default class Init extends Command {
         },
         {
           title: 'Installing dependencies',
-          task: () => installDependencies(projectDir, verbose).catch(e => this.error(`Error during npm install: ${e.message}`, { exit: 2 }))
+          task: () => installDependencies(projectDir, verbose).catch(e => this.error(`Error during npm install: ${e.message}`, { exit: 13 }))
```

Comment:
I had to google to know what 13 means. Can we put an enum somwhere? I didn't find a lib for that, but still...
https://github.com/nodejs/node-v0.x-archive/blob/master/doc/api/process.markdown#exit-codes

Reply:
Exit codes are just counting up from 10 (more or less) so we can identify where an error happened